File: php/control_flow/if_statement/if_else_multiple_statement.php
#!/usr/bin/php
<?php

if (1 > 2)
  echo "1 > 2\n";
else if (2 > 3)
  echo "2 > 3\n";
else
  echo "!(1 > 2) && !(2 > 3)\n";

if (1 > 2)
  echo "1 > 2\n";
else if (1 > 0)
  echo "1 > 0\n";
else
  echo "!(1 > 2) && !(1 > 0)\n";

if (2 > 1)
  echo "2 > 1\n";
else if (1 > 0)
  echo "1 > 0\n";
else
  echo "!(2 > 1) && !(1 > 0)\n";

echo "\n";

File: python/control_flow/if_statement/if_else_multiple_statement.py
#!/usr/bin/python3

if (1 > 2):
  print('1 > 2')
elif (2 > 3):
  print('2 > 3')
else:
  print('!(1 > 2) && !(2 > 3)')

if (1 > 2):
  print('1 > 2')
elif (1 > 0):
  print('1 > 0')
else:
  print('!(1 > 2) && !(1 > 0)')

if (2 > 1):
  print('2 > 1')
elif (1 > 0):
  print('1 > 0')
else:
  print('!(2 > 1) && !(1 > 0)')

print()
